docs/source/cli.rst

```diff
@@ -5,9 +5,9 @@
 Command Line Interface
 ======================
 
-When git-machete is installed it is added to your path, so it can be called from command line: ``git-machete <command> <options>``. git-machete comes with a wide variety of commands and their options and in this site you can find their documentation.
+When git machete is installed, it adds ``machete`` command to ``git``, so it can be called from command line: ``git machete <command> <options>``. git machete comes with a wide variety of commands and their options and in this site you can find their documentation.
```

Comment:
```suggestion
When git machete is installed, it adds ``machete`` command to ``git``, so it can be called from command line: ``git machete <command> <options>``. git machete comes with a wide variety of customizable commands and in this site you can find their documentation.
```